types/Package.js

```diff
@@ -69,18 +76,14 @@ const PackageMaintainerProps = S.object()
     'maintainers',
     S.array().items(PackageMaintainer)
   )
-  .prop(
-    'owner',
-    MaintainerId
-  )
 
 CompaniesSupportingPackage = S.array()
   .description('List of companies that are supporting a package')
   .items(
     S.object()
       .prop(
         'organizationId',
-        OrganizationId
+        S.string().maxLength(128)
```

Comment:
what was the reasoning behind this change ?

Reply:
importing anything from organization led to circular dependencies, wouldn't build

Reply:
we should probably have "id's" in a separate file that we can import from in each of these files